src/ui/data-models/list/DataModelListTableContainer.js

```diff
@@ -20,6 +23,9 @@ export const mapDispatchToProps = dispatch => ({
   onWillMount: (page) => {
     dispatch(fetchDataModelListPaged(page));
   },
+  onClickEdit: (dataModelId) => {
+    gotoRoute(ROUTE_DATA_MODEL_EDIT, { dataModelId });
```

Comment:
You could use gotoRoute or into thunks either set a route for a Link component

Reply:
this is injected at the moment